scripts/optimize_metaparameters.py

```diff
@@ -227,7 +230,7 @@ def GetObjfAndDeriv(x):
         # return negative infinity, and a zero derivative.
         print("Metaparameters not allowed: ", x)
         return (1.0e+10, np.array([0.0]*len(x)))
-
+   
```

Comment:
Plase be careful not to add whitespace like this.  My .emacs has this:
```
(add-hook 'write-file-hooks 'delete-trailing-whitespace)
```
(not sure if there is a vim equivalent).